baseplate/context/memcache/lib.py

```diff
@@ -0,0 +1,108 @@
+from six import string_types
+import logging
+import zlib
+
+
+try:
+    import cPickle as pickle
+except ImportError:
+    # python3
+    import pickle
+
+try:
+    from cStringIO import StringIO
+except ImportError:
+    try:
+        from StringIO import StringIO
+    except ImportError:
+        # python3
+        from io import StringIO
+
+
+FLAG_PICKLE = 1 << 0
+FLAG_INTEGER = 1 << 1
+FLAG_LONG = 1 << 2
+
+
+# python3 doesn't have a long integer class
+try:
+    long
+except NameError:
+    # python3
+    long = int
+    # don't override FLAG_LONG because we want to be able to deserialize values
+    # set by python2 processes. If a python3 process writes a large value as
+    # an integer a python3 process will still be able to deserialize it.
+
+
+def python_memcache_serializer(key, value):
+    flags = 0
+
+    if isinstance(value, string_types):
+        pass
+    elif isinstance(value, int):
```

Comment:
Seems like this isn't a problem actually:
```
>>> import sys
>>> sys.maxint
9223372036854775807
>>> int(sys.maxint + 100)
9223372036854775907L
```
python2 will automatically use `long` if the value is too big.